libs-scala/gatling-utils/src/test/resources/simulation-log/multiple-scenarios-expected-csv.csv

```diff
@@ -1,6 +1,6 @@
 simulation,scenario,maxUsers,request,start,duration,end,count,successCount,errorCount,min,p90,p95,p99,p999,max,mean,avg,stddev,rps
-foo,first,2,async,1000000000003,0.097,1000000000100,1,1,0,97,97,97,97,97,97,97,97,0,10.309278350515465
 foo,first,2,desync,1000000000004,0.096,1000000000100,2,1,1,95,95,95,95,95,95,95,95,0,10.416666666666666
+foo,first,2,async,1000000000003,0.097,1000000000100,1,1,0,97,97,97,97,97,97,97,97,0,10.309278350515465
```

Comment:
So this was apparently necessary. I'm guessing some collection semantics changed.

@S11001001, can you tell me if this change is reasonable or if we should be getting to the bottom of it?

Reply:
should be fine I think. We don’t depend on the exact results here. My suspicion is that associativity changed but dunno.